0019-glossary/0019-glossary.md

```diff
@@ -0,0 +1,272 @@
+asset. This way, two strangers can run a ledger with properties similar to a trustline, but without fully trusting each other.
+
+## Peer balance
+See [trustline](#trustline).
+
+## Peering
+Connectors "peer" with one another to exchange information used to determine the best route for a payment, and to route it (connect its chain of
+transfers through [forward](#forward-action) and [backward](#backward-action) action.
+A peering relationship is usually implemented using either a [trustline](#trustline) or a [payment channel](#payment-channel)..
+
+## Prepared
+The second state of an interledger transaction, after [proposed](#proposed), and
+before its [final state](#final-state).
+
+## Pre-Shared Key (PSK)
+The recommended transport layer for most use cases.
+In Pre-Shared Key (PSK) protocol, the sender and receiver use a shared secret to generate the payment condition, authenticate the ILP packet, and encrypt application data. Using PSK, the sender is guaranteed that fulfillment of their transfer indicates the receiver got the payment, provided that no one aside from the sender and receiver have the secret and the sender did not submit the fulfillment. See [IL-RFC 16](../0016-pre-shared-key/0016-pre-shared-key.md).
+
+## Proposed
+The initial state of an interledger transaction, before [prepared](#prepared).
+
+## Receiver
+The party who is the beneficiary (payee) of a transfer or a transaction (payment).
+
+## Route
+A list of [transfers](transfer) and the [hops](#hop) between them.
+
+## Routing table
+A lookup table, used by a connector to decide who their outgoing receiver should be (the next
+hop, which may be another connector, or the destination receiver), and what the lowest outgoing
+amount is which they can safely offer for the outgoing transfer (equal to the destination amount in case
+the outgoing is the destination transfer).
+
+## Sender
+The party who is the initiator (payer) of a transfer or a transaction (payment).
+
+## Simple Payment Setup Protocol (SPSP)
+SPSP uses Webfinger ([RFC 7033](https://tools.ietf.org/html/rfc7033)) and an HTTPS-based protocol for communicating account, amount, and Pre-Shared Key details from the receiver to the sender.
+See [IL-RFC 9](../0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md).
+
+## Source (transfer, ledger, amount)
+The transfer/ledger/amount directly adjacent to the sender.
+
+## Transaction
+The movement of assets from one party to another. A transaction consists of one or more [ledger transfers](#transfer). Roughly synonymous with
+[payment](#payment).
+
+# Transfer
+The movement of assets on one single ledger. Multiple transfers can be chained together into one multi-hop [transaction](#transaction).
+Also sometimes called 'ledger transfer'.
+
+## Transport layer
+The middle layer of the Interledger protocol stack, which can be either
```

Comment:
It's most important to describe the layer in terms of what it does. In this case it's determining the condition and encoding of the data in the ILP packet, and what details the sender and receiver need to discuss beforehand